content/docs/actions.md

```diff
@@ -73,47 +75,81 @@ archive. If there are more steps, then one of them has to return the archive
 name. The best practice is to do it from the last step and print it: `bash -c
 'echo path/to/archive-$VERSION.tar.gz'`.
 
+If you can, please place the generated archive in the same directory as your
+spec file.
```

Comment:
btw, it seems it is not needed with packit v0.35+ (tested with sources in a subdir).

Reply:
for single-source projects, it's true it's not needed but preferred since it may save you some trouble when generating it

hence the wording "do it if you can" :)

Reply:
even with multiple sources :) but you are right, the "if you can" is good.